client/signup/steps/domains/index.jsx

```diff
@@ -1330,9 +1357,11 @@ export class RenderDomainsStep extends Component {
 		if ( this.props.step && 'invalid' === this.props.step.status ) {
 			content = (
 				<div className="domains__step-section-wrapper">
-					<Notice status="is-error" showDismiss={ false }>
-						{ this.props.step.errors.message }
-					</Notice>
+					{ this.props.step && 'invalid' === this.props.step.status && (
```

Comment:
Curious about what this line does, because on line 1357 it checked `this.props.step && 'invalid' === this.props.step.status` so doesn't this always resolve to `true`?

Reply:
Good catch, thanks! That's dead code. 😅